schematic/__init__.py

```diff
@@ -20,6 +20,7 @@
 # Suppress INFO-level logging from some dependencies
 logging.getLogger("keyring").setLevel(logging.ERROR)
 logging.getLogger("rdflib").setLevel(logging.ERROR)
+logging.getLogger("Synapse storage").setLevel(logging.INFO)
```

Comment:
Does the synapse Store need to be hardcoded to the `INFO` log level?

Reply:
Hmm.. so I wanted to be able to log the part of purging AWS cache on AWS cloudwatch console: 
```
logger.info(f'{num_of_deleted_files} number of files have been deleted from {root_dir}')
```
and
```
logger.info(f'Estimated {remaining_space} bytes (which is approximately {converted_space}) remained in ephemeral storage after calculating size of .synapseCache excluding OS')
```
But I could also change `logger.info` to `logger.debug` if that's better.

Reply:
That makes sense that you'd want to log the cache purging. I was just concerned about hard coding the logging level for the whole store when schematic is initialized. Is there a way to set the logger when you're running it on AWS so that it isn't changed for other contexts or users?

Reply:
hmm I don't think there is such a way especially that we are going to use `develop` for deployment